Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "Installer launch sample" 
+page_type: sample
+languages:
+- cpp
+products:
+- windows
+- windows-api-win32
+description: "Shows how to use CreateProcess to launch the installer without a console window."
+urlFragment: installer
+---
+# Installer launch sample
+
+This sample demonstrates how to launch the WindowsAppRuntimeInstall.exe without having a pop-up
+window appearing while it is running. This sample is intended for Win32 programs which have setup
+programs that need to embed the WindowsAppSDK without any pop-up windows.
+
+## Requirements
+Sample requires downloading or building a version of WindowsAppRuntimeInstall.exe. The sample
+assumes that it will be in %USERPROFILE%\Downloads for convenience. If a different path is
+desired, it can be specified in the #define at the top in main.cpp.
```

Comment:
Maybe add a link to the docs for where you can download WindowsAppRuntimeInstall.exe

Reply:
Can we add a link to the Downloads page to get the installer: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads